packages/kit/test/apps/basics/src/routes/store/client-access/+page.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
 	import { getStores } from '$app/stores';
 
-	let pathname;
+	let pathname = 'undefined';
```

Comment:
is there a reason to do this as opposed to changing line 7 to what you did elsewhere:
```
{`${pathname}`}
```

it might be nice to be consistent in the way we address this

Reply:
Agreed. I've updated it to use the string template.